server/src/cloud/board.ts

```diff
@@ -67,6 +67,7 @@ const respondToJoinRequest = async (
 export interface CreateBoardRequest {
   columns: {
     name: string;
+    color: string;
```

Comment:
Why don't you use the Color type from `src/constants/colors.ts` instead of string?